CONTRIBUTING.md

```diff
@@ -57,7 +57,7 @@ You can run `tox` with the following arguments:
 - `tox -e py312-opentelemetry-api` to e.g. run the API unit tests under a specific
   Python version
 - `tox -e spellcheck` to run a spellcheck on all the code
-- `tox -e lint` to run lint checks on all code
+- `tox -e lint-some-package` to run lint checks on `some-package`
 
 `black` and `isort` are executed when `tox -e lint` is run. The reported errors can be tedious to fix manually.
 An easier way to do so is:
```

Comment:
1. Run `.tox/lint-some-package/bin/black .`
2. Run `.tox/lint-some-package/bin/isort .`

Reply:
I'm trying to be [consistent](https://github.com/open-telemetry/opentelemetry-python-contrib/blame/main/CONTRIBUTING.md#L74).